### ISSUE TITLE
Parallelized build failure: No rule to make target `y.tab.h'

### ISSUE THREAD

**Issue by ilovezfs** [elboza/dwarf-ng#3]
```
==> make install
Making install in src
Making install in libdwarf
bison -y -y -d -v -p cfgyy cfglang.y -o y.tab.c
flex -P cfgyy cfglang.l
make[2]: *** No rule to make target `y.tab.h', needed by `install'.  Stop.
make[2]: *** Waiting for unfinished jobs....
conflicts: 2 reduce/reduce
make[1]: *** [install-recursive] Error 1
make: *** [install-recursive] Error 1
```

This hack prevents the issue, but really a proper Makefile.am is needed:

```
make -C src/cmdline y.tab.c
make -C src/libdwarf y.tab.c
```

This bug was also mentioned in #1

**Comment by ilovezfs** [comment]
@elboza ping on this.

**Comment by elboza** [comment]
solved with dawrf-0.3.2

**Comment by ilovezfs** [comment]
@elboza I have reinstated dwarf in Homebrew: https://github.com/Homebrew/homebrew-core/pull/15058


Thanks again for the fixes and the upgrade!
